pkg/api/api.go

```diff
@@ -142,3 +177,44 @@ func (dvol *DvolAPI) AllVolumes() ([]string, error) {
 	}
 	return volumes, nil
 }
+
+func (dvol *DvolAPI) Commit(activeVolume, activeBranch, commitMessage string) (string, error) {
+	commitId, err := dvol.dl.Snapshot(activeVolume, activeBranch, commitMessage)
```

Comment:
In absence of a type defining what this returns, please document what it returns as you would do in Python.

Really though I think a `type Commit { CommitID string}` is in order.